recipes/tests.rb

```diff
@@ -443,6 +443,32 @@ module load intelmpi && mpirun --help | grep '#{node['cfncluster']['intelmpi']['
   end
 end
 
+# Verify FabricManager version is aligned with Nvidia drivers
+case node['cfncluster']['os']
+when 'alinux2', 'centos7', 'centos8'
```

Comment:
minor, instead of listing OS, you could have used
```
case node['platform_family']
when 'rhel', 'amazon'
   [...]
when 'debian'
   [...]
```

Reply:
done